bot.js

```diff
@@ -220,13 +220,15 @@ process.on('unhandledRejection', error => {
 /**
  * Errors Handler.
  * @param {Error} error The catched error
- * @param {Discord.Message} ctx Discord message with context
+ * @param {Discord.Message} [ctx] Discord message with context
  * @async
  */
 async function onError(error, ctx) {
-	// Gets the language
-	const lang = await ctx.bot.getLanguage(ctx);
+	// Gets the language.
+	let lang = 'en';
+	if (ctx && ctx.bot) lang = await ctx.bot.getLanguage(ctx);
```

Comment:
@mprangenberg 
I still think it might be risky to translate error messages.
If somehow the `getLanguage(ctx)` method throws an error, the bot is stuck into a log loop until it crashes due to missing drive space (last loop error generated 200Mb of logs in 2 minutes).

Reply:
Yeah, i fully agree with you. But most of the things translated down here are important to translate. In my opinion they shouldn't be handled as an error here, but should already be handled in their respective classes. But i didn't want to do a complete overhaul because i was more focused on translating.

Reply:
I think at least the following three could be handled on the spot, either by try...catch or just a ctx.reply. If you need the console output maybe just put it there, too.
- TooManyDiceError
- NoSelectionElementsError and 
- SelectionCancelledError

What do you think?

Reply:
I think, to be safe, I will put the line `const lang = await ctx.bot.getLanguage(ctx);` into a try..catch.

I have put TooManyDiceError, NoSelectionElementsError and SelectionCancelledError here because I wanted to avoid repetitive code as these errors can be thrown by several different commands and are handled the same way.

Reply:
The try...catch is the safest way to do it, good idea.

If you search through the project, these specific errors are being thrown in one line each, never in multiple places. So there wouldn't be repetitive code. Unless i overlooked something.

Reply:
You are totally right! ..for now!
In the future, I might throw these errors from other places.